### PR TITLE
Added line 35 to point new contributor to Contribute to Kubernetes docs link.

### LINKED ISSUE #20161: Link to “Start Contributing” from CONTRIBUTING.md
**This is a Feature Request**

**What would you like to be added**
Revise https://github.com/kubernetes/website/blob/master/CONTRIBUTING.md to signpost new contributors to https://kubernetes.io/docs/contribute/start/

**Why is this needed**
If someone arrives at https://github.com/kubernetes/website/blob/master/CONTRIBUTING.md (possible from https://github.com/kubernetes/website/contribute) and has not already contributed to the website, then it's good to let them know that there is a whole section of documentation aimed at exactly that target audience.

**Comments**
/language en
/good-first-issue

### PR DESCRIPTION
**Pull request by Heena-PyCloud**
Added line 35 to point new contributor to Contribute to Kubernetes docs link(https://github.com/kubernetes/website/contribute).

Closes #20161

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -32,6 +32,8 @@ Kubernetes.io uses github issues to track documentation issues and requests. If
 
 Note that code issues should be filed against the main kubernetes repository, while documentation issues should go in the kubernetes.io repository.
 
+If you’re just getting started, consider exploring this resource [Contribute to Kubernetes docs](https://kubernetes.io/docs/contribute/).
```
Comment (zacharysarah) [inline CONTRIBUTING.md:35]:
```suggestion
For more information, see ["Contribute to Kubernetes docs"](https://kubernetes.io/docs/contribute/).
```

**Comment by tengqm** [comment]
@zacharysarah

**Comment by zacharysarah** [comment]
@Heena-PyCloud Thanks for your patience, and sorry for the long delay.

Please reword your commit messages to exclude GitHub keywords. 

Screen Shot 2020-07-08 at 6 55 55 PM

**Comment by jimangel** [comment]
@Heena-PyCloud are you still interested in working on this? After 14 days a PR wrangler might close this PR if no action is taken.

**Comment by kbhawkey** [comment]
Hello @Heena-PyCloud . 
This PR is very close to merging. Are you able to modify the commit message to remove the text,
```Closes #20161```?

**Comment by kbhawkey** [comment]
Hello @Heena-PyCloud .
I plan on closing this pull request because the PR cannot be merged without changing the commit message.
Would it be possible for you to open a new PR with these changes?
We appreciate your contributions! Thanks.
/close

**Pull request closed without merging**